# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsPageFrame.cpp
# Commit: a261bb633790
# Full Hash: a261bb633790ddca9cc8ce83adca0a1bb36114a6
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2022-02-23 09:38:00
# Regressor Bug: 1714584
# File Overlap Count: 2
# Description:
#   Bug 1714584 - Part 1: Decouple nsDisplayList internal list from nsDisplayItems r=mstange
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138152
# ==============================================================================

diff -r 635f402efb1b -r a261bb633790 layout/generic/nsPageFrame.cpp
--- a/layout/generic/nsPageFrame.cpp	Tue Feb 22 23:42:08 2022 +0000
+++ b/layout/generic/nsPageFrame.cpp	Tue Feb 22 23:42:18 2022 +0000
@@ -585,7 +585,7 @@
 
 void nsPageFrame::BuildDisplayList(nsDisplayListBuilder* aBuilder,
                                    const nsDisplayListSet& aLists) {
-  nsDisplayList content;
+  nsDisplayList content(aBuilder);
   nsDisplayListSet set(&content, &content, &content, &content, &content,
                        &content);
   {